### ISSUE TITLE
Must implement the User favorite games list

### ISSUE THREAD

**Issue by HeinzDev** [HeinzDev/Timothy-Project#6]
the user must have an page with the games he want to play (Starred games).

**Comment by HeinzDev** [comment]
The code in the express file will be something like:

```
app.post('/users/:user/games/:game/star', async (req, res) => {
  const userId = req.params.user;
  const gameId = req.params.game;

  try {
    await GamesModel.updateOne({ _id: gameId }, { $inc: { stars: 1 } });

    const user = await UserModel.findById(userId);
    user.starredGames.push(gameId);
    await user.save();

    res.status(200).send('Starred!.');
  } catch (error) {
    res.status(500).send('error favoriting the game: ' + error);
  }
});

app.get('/users/:user/games', async (req, res) => {
  const userId = req.params.user;

  try {
    const user = await UserModel.findById(userId).populate('starredGames');
    const favoriteGames = user.starredGames;

    res.status(200).json(favoriteGames);
  } catch (error) {
    res.status(500).send('Couldn't get the favorite games: ' + error);
  }
});
```

Problably will only be implemented after the branch aimed to build the UI is finished

**Comment by HeinzDev** [comment]
The endpoint was implemented in commit d165540ca3954d8c4be74f010d44ad2f4189e72c. But this issue needs the frontend to be implemented first, and tests with this functionality must be made too

**Comment by HeinzDev** [comment]
Working in progress⚙️
